src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/controller/GoalController.kt

```diff
@@ -86,7 +86,7 @@ class GoalController(private val service: GoalService) {
     @RequestBody reAddGoal: Goal,
   ): GoalEntity {
     reAddGoal.status = if (reAddGoal.targetDate.isNullOrEmpty()) GoalStatus.FUTURE else GoalStatus.ACTIVE
-    return service.updateGoalStatus(goalUuid, reAddGoal)
+    return service.reAddGoal(goalUuid, reAddGoal)
   }
 
   @PatchMapping("/{goalUuid}")
```

Comment:
I think that this endpoint can now come out

Reply:
Removed in [1e4b4d7](https://github.com/ministryofjustice/hmpps-sentence-plan/pull/199/commits/1e4b4d76850be0d3b55bc82a5a91be5f74093f2c)